src/ansys/fluent/core/session.py

```diff
@@ -164,6 +164,8 @@ def _build_from_fluent_connection(
     ):
         """Build a BaseSession object from fluent_connection object."""
         self._fluent_connection = fluent_connection
+        # Stores the backup once fluent connection is nullified.
```

Comment:
@prmukherj Why isn't the bakup always valid? It would be less complicated that way.

Reply:
ok, sounds right. This will answer your other point as well.